inject_test.go

```diff
@@ -1,8 +1,7 @@
-package inject_test
+package inject
```

Comment:
This is for a different reason. In the original code, it's referring to the inject package using absolute path "github.com/codegangsta/inject". With the fork we're not able to do that anymore (otherwise the test is not runnable).